forge-gui/res/cardsfolder/upcoming/temporary_lockdown.txt

```diff
@@ -0,0 +1,7 @@
+Name:Temporary Lockdown
+ManaCost:1 W W
+Types:Enchantment
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigExile | TriggerDescription$ When CARDNAME enters the battlefield, exile each nonland permanent with mana value 2 or less until CARDNAME leaves the battlefield.
+SVar:TrigExile:DB$ ChangeZoneAll | ChangeType$ Permanent.nonLand+cmcLE2 | Duration$ UntilHostLeavesPlay | Origin$ Battlefield | Destination$ Exile
+SVar:OblivionRing:TRUE
+Oracle:When Temporary Lockdown enters the battlefield, exile each nonland permanent with mana value 2 or less until Temporary Lockdown leaves the battlefield.
```

Comment:
Something tells me the AI isn't going to use this very well... most cards like this target or more specifically lock down just the opponent. Could you toss this in an AI deck and see if it uses it wisely? Maybe set up the AI opponent where they're ahead on board due to weenies. If the AI casts this even when it isn't advantageous, we can put `AI:RemoveDeck:All` on it and make an issue for later.